wasm_for_tests/tx_proposal_ibc_client_upgrade/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
Seems to me like there are a bunch of dependencies that might not be needed, but this is something that extends to all the wasm for tests, I'll open an issue to track this

Reply:
Tracked here #3606